src/sidebar/components/menu-item.js

```diff
@@ -55,23 +55,33 @@ export default function MenuItem({
   }
   const leftIcon = isSubmenuItem ? null : renderedIcon;
   const rightIcon = isSubmenuItem ? renderedIcon : null;
+  // Only add `aria-checked` attribute if `isSubmenuItem` is false. Ensure that the value
+  // is a "true" or "false" string
+  const ariaChecked = !isSubmenuItem
```

Comment:
This got a bit tricky but what I wanted to end up with there is only adding the prop when it had the role="radio"
and I wanted the value to be a string, not a bool. I added corresponding tests for each case